root/layout/components/Footer.js

```diff
@@ -26,8 +26,8 @@ component Footer() {
         <a className="internal" href={DONATE_URL}>{l('Donate')}</a>
         <a className="internal" href="//wiki.musicbrainz.org/">{l('Wiki')}</a>
         <a className="internal" href="https://community.metabrainz.org/">{l('Forums')}</a>
-        <a className="internal" href="/doc/Communication/IRC">
-          {l('Chat (IRC)')}
+        <a className="internal" href="/doc/Communication/ChatBrainz">
+          {l('Live chat')}
```

Comment:
Live chat kinda makes me think of those customer service popups on every website. 😅

Reply:
I know - maybe just "chat" is good enough? I wasn't sure what to call it, I wanted to make it clear it's a place to talk synchronously as opposed to the forum.

Reply:
"Chat" seems fine. My other thought was "real-time chat," which might sound too technical (though not nearly as much as "IRC" :)).